x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/SqlVersion.java

```diff
@@ -74,6 +76,44 @@ public static SqlVersion fromString(String version) {
         return new SqlVersion(version, from(version));
     }
 
+    /**
+     * Parses a transport version string into a {@link SqlVersion}.
+     * <p>
+     *     <b>Only use for testing.</b>
+     * </p>
+     * <p>
+     *
+     * The transport version string can take the following shapes:
+     * <ul>
+     *     <li>Major.Minor.Revision</li>
+     *     <li>Major.Minor.Revision-Major.Minor.Revision</li>
+     *     <li>Major.Minor.Revision-snapshot[Id]</li>
```

Comment:
This is modified by https://github.com/elastic/elasticsearch/pull/112242

Reply:
Thanks, adjusted.

Reply:
This has been removed entirely now.